change/@fluentui-react-tabs-7a95002f-25eb-47c0-9f9b-a77ffdef753a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33441&commitId=3ede9f4135f57855bd90b77ae8cce8001e91b4ad&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|20| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|7| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2232| Changed|
|Drawer.overlay drawer full.chromium.png|1155| Changed|

</details>


  <details><summary><b>TabList and Tab Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TabList and Tab Converged.Filled Circular Appearance Dark Mode.chromium.png|19608| Changed|
|TabList and Tab Converged.Subtle Circular Appearance Dark Mode.chromium.png|19527| Changed|
|TabList and Tab Converged.Filled Circular Appearance.chromium.png|806| Changed|
|TabList and Tab Converged.Filled Circular Appearance High Contrast.chromium.png|23073| Changed|
|TabList and Tab Converged.Subtle Circular Appearance.chromium.png|570| Changed|
|TabList and Tab Converged.Subtle Circular Appearance High Contrast.chromium.png|23660| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>